src/geodesic.rs

```diff
@@ -260,7 +260,7 @@ impl Geodesic {
                 J12 = m0x * sig12 + (A1 * B1 - A2 * B2);
             }
         } else if outmask & (caps::REDUCEDLENGTH | caps::GEODESICSCALE) != 0 {
-            for l in 1..self.GEODESIC_ORDER {
+            for l in 1..=self.GEODESIC_ORDER {
```

Comment:
Right. I'm satisfied that my changes so far move in the right direction, but it looks to me like it will take at some more work (I suspect at least 2-3 weeks at my currently-reduced time commitment) to establish a match with either the GeodTest S12 value or the C++ S12 value for most rows in GeodTest.dat. I'm guessing I'll carve out at least a handful of cases for later work under a new issue, but I'd like narrow it further than I have so far.